docs/connect.rst

```diff
@@ -18,9 +18,8 @@ class so we can use it to connect:
 
     from bigchaindb_driver import BigchainDB
 
-Finally, to make the connection, you need to know
-the BigchainDB "API Root Endpoint"
-of a BigchainDB node or cluster.
+Finally, to make the connection, you need to know the URL address of the
```

Comment:
The BigchainDB HTTP API docs call this (e.g. `http://example.com:9984`) the "BigchainDB Root URL", e.g. see https://docs.bigchaindb.com/projects/server/en/master/drivers-clients/http-client-server-api.html#bigchaindb-root-url